geocity/apps/reports/templates/reports/report.css

```diff
@@ -1,16 +1,5 @@
 @page {
   font-family: "{{report.layout.font}}";
-  /* meta content (see https://www.quackit.com/css/at-rules/css_top-left-corner_at-rule.cfm) */
-  @bottom-right-corner {
-    content: counter(page);
-    display: flex;
-    align-items: center;
-    justify-content: center;
-  }
-
-  @bottom-center {
-    content: "{% now "DATETIME_FORMAT" %}";
-  }
```

Comment:
Create block and find a way to add in python a counter, date time, and more.. https://www.quackit.com/css/at-rules/css_top-left-corner_at-rule.cfm (keep this in comment)